### DISCUSSION TITLE
The new mobile UI needs some more improvements to really be mobile/touch-friendly

### DISCUSSION THREAD

**Discussion by immstms** [standardnotes/forum#3114] [Suggestions]
Plzzzzzzz!! Bring Version 3.41.10(237) back!!!!
The new version of 3.43.5 is totally sucked!!!!

**Comment by effieeee** [comment]
Hi @immstms I'm afraid the new interface is really set to replace the old UI, though perhaps you can share with us more details about the things you don't like on the new version? We can certainly try to work on those to help improve it further!

**Comment by madameblueberryxo** [comment]
I made an account here just to come and say this same thing. I greatly prefer the old UI. 

I prefer the old log in screen and I prefer how toggling the menu on the left and the individual note options on the right still left a quarter of the screen visible. And I liked how I could swipe the menu or the options away instead of having to press something to get back to where I was.
I liked how in the old menu looked and how my email and account was at the top with the setting gear at the bottom.
I liked how the "+" for the new note was on the bottom right corner.
I liked how in the Options menu there was a clear tag button in the bottom right for a new tag.
I liked how on mobile creating a new note just left the title blank rather than having it automatically fill in with the date.
I liked how there was no menu bar at the top of each note, just the blank title box, back button and the hamburger in the upper right for the Options.
I liked how there was more of a preview of each note in the old version and how each note in the notes list was thicker vertically with the two lines of preview instead of one.
I massively preferred the font in the notes on the old version too. The old font was more in line with the rest of my iPhone and seemed smaller and I really really liked that.
I liked how the search box was centered at the top of the screen in the old version and that it was more square than round.
I liked how there was no left-hand margin as I was writing my notes.
I liked how the old version didn't have a pop up about applying the system color scheme in 5 seconds.

I could go on.
Just about everything that's different now, I liked it better before. What I really miss is the old note font and not having that left-hand margin and the note title automatically being left blank. 
Standard Notes is my most used iPhone app, I use it daily, and I've used it for years and I've greatly enjoyed it. I use it like I'm talking to a therapist and it has helped me so much. I love Standard Notes and I love the old UI even more. I have thought about upgrading in the past but now I am glad I didn't because if I had been a premium user already, the new UI would have made me regret it. It would be nice if there was some way to downgrade to the old version or, if getting back to the old UI was a premium feature, I would gladly make the upgrade.

**Comment by xthrk** [comment]
I don't think the Version 3.43.5 is sucked. I really like the fact that the range of functions is fully covered with the New Mobile Experience, but there is space for improvement, because the handling and design are not nice. I know the team is working hard on it, so here is some feedback:

Navigation and operation overall has become more cumbersome, not comfortable and not intuitive. Design is not consistent: sometimes the back button (<) is at the top if you are in a note, for example, or at the bottom if you have the burger menu (Views; Notes, etc. and Folders) open.

The New Mobile Experience is not touch-friendly. The buttons don't always respond, possibly I don't always hit them because they are too small. This does not make it possible to navigate quickly.

The button for a new note are now on the top right. It has become very cumbersome to use (especially on large smartphones) as you have to reach around frequently. Also, you have to search again and again where a currently needed button is located; sometimes top right, then top left, then bottom left, bottom right, then top center, depending on what you need at the moment.

With long notes, if you want to get to the header (note menu, tags, files, etc.), you have to scroll up again for a very long time.

I use mostly plaintext notes on the desktop. The font of the plaintext notes in the New Mobile Experience is smaller, narrower and thus very hard to read.

I need more clicks to perform certain actions, for example selecting a tag.

Long pressing on a note in the overview opens the menu to the note, but then it doesn't show for which note I opened the note menu (note topic missing). So I don't know if I actually got the correct note.

**Comment by effieeee** [comment]
> The New Mobile Experience is not touch-friendly [or mobile friendly]

I think this nicely sums up a lot of the feedback we've been seeing so far about the new UI, which is understandable because the new UI at its current state is a mobile rework of the current web UI, with some functions not quite fully optimized or adapted for mobile use just yet. Thanks everyone for sharing your thoughts so far! 🙏

**Comment by snfeedback** [comment]
I love Standard Notes and use it everyday.  As a long time user of Standard Notes on iOS (iPhone), the most relevant frustrating UI change is that adding tags to notes is much more cumbersome - just enough to make filing a quick note annoying enough to not want to do it.  Whatever the old system was worked very smoothly.  If you have many tags, you have to scroll down a list of tags then manually select the desired tag then press arrows to return to the note.  The process is long enough to distract me from the thought that I wanted to capture in the first place.  A critical function of this program is to be able to take notes.  Filing notes that are not organized by tags is a big usability issue for me.  It feels like the app is getting in the way of what I want to do with it.  Ideally, adding tags should be possible from note page without leaving to another screen... ideally by starting to type in the first letters of the tag like the desktop app.  If one thing needs to be change to make this usable again for me, this would be it.  I also created a GitHub account just to make this comment.  I hope the team can prioritize usability of tags on mobile/iOS.

Thanks.

**Comment by xthrk** [comment]
When I want to create a new note on my mobile device, the cursor is not in the text/editor area, but in the heading. So you always have to type again (to set the cursor in the editor area) additionally if you want to enter a note.

**Comment by noodlejetski** [comment]
> though perhaps you can share with us more details about the things you don't like on the new version? 

the new interface fits the operating system's UI even less than the old version. there's a complete lack of animations, which makes the app feel clunky and unfinished. the side panel not being accessible by swiping means that I now have to reach to the top of my phone's screen when I want to access it. compared to Android's Material You and its recent focus on consistency and reachability (important buttons moved to the bottom of the screen for easy access, apps adapting their colours to the device's theme, and so on), the entire app feels severely unpolished and sticks out like a sore thumb, to the point where I'm considering looking for alternatives after having used (and recommended to other people) Standard Notes for years.

**Comment by eloops** [comment]
I can only agree with comments here. For something I paid money for, the rollout of the new UI (I use it on iOS) is awful. There's no option to revert to the old UI. There was no notification that there might be a new UI being rolled out, nor an option to opt in or out of it.

I expect these sort of wholesale changes from a larger company offering a free application. Why is this being shoved onto paying customers?

As for what is wrong with it:

- Copy and paste behaviour is really inconsistent. It's difficult to highlight the correct text. In other applications this is not a problem
- When the keyboard pops up, it covers the bottom of the document. I'm unable to swipe any where to make this visible.
- "Buttons" for navigation have been moved from previous positions and move around the screen depending on where you are in a note

Go back and re-read your note on longevity and tell me this UI update reflects your own values. https://standardnotes.com/longevity

**Comment by ghost** [comment]
I agree with the above.  I just updated manually on Android (big mistake) after reading this post (https://standardnotes.com/blog/react-native-is-not-the-future) and now I am questioning stopping my subscription because I just hate being forced away from something that was comfortable to use to the current UI.

While the old version of the Android app did often crash mid-typing on larger notes (too much data to sync?), I was happy with the usable button sizes and small/space-efficient but still viewable text.

On the other hand, the summary of my complaint with the new app is that the buttons are too small (not optimized, or seemingly any consideration given for mobile devices), and the text is too large (can't read enough in note previews, and the amount of text-on-screen within a note has decreased).  Also, the "drawer" side bar opens too far over (the entire screen now), whereas before it was only half or less of the screen (which was sufficient and easier to close).

It seems to me that the devs decide to do something to reduce app complexity for themselves and make their lives easier (which is generally a good thing), while compromising the user experience (which should always be first priority, especially for paying customers).  At minimum, a rollback to the previous version of the app should have been added as an option.

**Comment by nevadaperry** [comment]
I noticed that at least two of the issues were resolved recently, so thank you to the team for that.
* Swiping from the left edge to go back o the main notes list now works again.
* There's now a button to hide the keyboard while editing.

**Comment by wenman-ualberta** [comment]
Does the Standard Notes code framework support plug-ins that would change the UI?

I would like to program a plug-in that does that, first for myself, & then to share with others.

I would publish it under some kind of FLOSS license so Standard Notes would be legally and financially free to adopt it officially if it is well received.
